runelite-client/src/main/java/net/runelite/client/plugins/emojis/EmojiPlugin.java

```diff
@@ -166,6 +225,54 @@ public void onOverheadTextChanged(final OverheadTextChanged event)
 		event.getActor().setOverheadText(updatedMessage);
 	}
 
+
+	@Subscribe
+	public void onGameTick(GameTick gameTick)
+	{
+		timeoutEmojiMessages(false);
+	}
+
+	@Subscribe
+	public void onMenuOptionClicked(MenuOptionClicked event)
```

Comment:
This should subscribe to `WidgetMenuOptionClicked` now that MenuManager is being used to add the menu options. Similarly, this should guard on both the menu option text (like it currently does) and that the widget is one of the all tabs. I would reference other plugins making use of MenuManager such as the inventory tags and instance map plugins to see similar usages there.

Ex:

```java
private static Set<WidgetInfo> ALL_TABS = ImmutableSet.of(
	FIXED_INVENTORY_ALL_TAB,
	RESIZABLE_INVENTORY_ALL_TAB
);

@Subscribe
public void onWidgetMenuOptionClicked(WidgetMenuOptionClicked event)
{
	if (ALL_TABS.contains(event.getWidget())
		&& event.getMenuOption().equals(EMIT)
		&& event.getMenuTarget().equals(MENU_TARGET))
	{
		timeoutEmojiMessages(true);
		addEmojiPaletteMessage();
	}
}
```

Creating an `ALL_TABS` collection would have the benefit of de-duplicating the menu manager calls later on as well.